### ISSUE TITLE
Syndicate dartgun

### ISSUE THREAD

**Issue by ShadowDeath6** [yogstation13/yogstation-classic#356]
Stealing shit from goon again 2- this time it's not personal

Basically adds a mini syringe gun that you can load a beaker of chems in, and it will produce an infinite amount of syringes as long as you have chemicals to shoot from it. Probably would cost like 4 or 5 tc and almost exclusively used in conjunction with the toxins kit. Also used by chemists (B1g) that want to kill people without using patches or a big clunky syringe gun.

**Comment by Stealthkibbler** [comment]
Not necessary, syndicate shuttle already comes equipped with a syringe gun and you can order toxin kit and bioterror syringes and the nanomed also dispenses syringes.

**Comment by X-TheDark** [comment]
He probably meant for regular traitors, not nuke ops. And no, another ezpz combo cafe (Toxin+Syringe in one package)? Against.

**Comment by ghost** [comment]
retarded no

**Comment by ToGWtF** [comment]
You can get a rapid syringe gun with 5 minutes effort in R&D

**Comment by ghost** [comment]
Let me explain why in detail.

Because it is RETARDED that people are so completely incompetent as any form of antagonist they depend on their uplink in any possible way for their shit because LOL, WHAT IS ACTUALLY BEING INTELLIGENT?

Why the FUCK are you making it easy peasy for traitors? It's another combo fucking cafe for a mode that doesn't need one. You want a syringe gun or a method of applying your poison to people? Think of it yourself, that's the point behind the poison kit.

You are SUPPOSED to use station gear to equip yourself. Syringe gun, spray bottle, ect.

Giving traitors yet another easy fucking way out of anything is retarded. Infinite syringes, are you fucking drooling?

**Comment by B1gb3ast** [comment]
Wasn't the point of traitor that your uplink safes you the need of doing things like RnD?

You could get a taser, or you could get an easy tc lemon squeazy and buy an energy bow.

You could do chem and make super lethal things, or you could get an easy tc lemon squeazy and buy a toxins kit.

You could learn wires and get a filled up toolbelt, or you could get an easy tc lemon squeazy and buy a crypto.

@Adam-el Tablawy If you're really going to argue that traitors shouldn't get gear they can already get in the first place, then I have no clue what you are argueing in the first place.

I do think this item is neat, but there might be some balance issues. As is I can fill my backpack up with 15 syringes and a syringe gun, this'd safe so much space that it might allow me to pack too much of an arsenal.

I find it laughable that people are calling a syringe gun with syringes a wombocombo, how soft can you get?

**Comment by B1gb3ast** [comment]
Tl;dr: item would be redundant and mainly for lazy traitors.

**Comment by ghost** [comment]
Close this, feedback is negative.
